client/question/services/FeedbackGeneratorService.js

```diff
@@ -594,6 +596,46 @@ tie.factory('FeedbackGeneratorService', [
       }
     };
 
+    /**
+     * Returns the Feedback object associated with a compilation submission.
+     *
+     * @param {string} errorString A string detailing any errors.
+     * @returns {Feedback}
+     * @private
+     */
+    var _getCompileFeedback = function(errorString) {
+      // If we get a timeout error, here, it's not due to the student's code.
```

Comment:
I think the timeout error should not be sent to the frontend at all if it's not actually a timeout error.